stac/open-skysat-data/20201213_045438_ss01_u0001/20201213_045438_ss01_u0001.json

```diff
@@ -1,21 +1,24 @@
 {
   "stac_version": "1.0.0",
   "stac_extensions": [
-    "https://stac-extensions.github.io/eo/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/eo/v1.1.0/schema.json",
     "https://stac-extensions.github.io/view/v1.0.0/schema.json",
-    "https://stac-extensions.github.io/projection/v1.0.0/schema.json"
+    "https://stac-extensions.github.io/projection/v1.1.0/schema.json",
+    "https://planetlabs.github.io/stac-extension/v1.0.0-beta.2/schema.json"
   ],
   "type": "Feature",
   "id": "20201213_045438_ss01_u0001",
   "collection": "planet-stac-skysat",
   "properties": {
     "datetime": "2020-12-13T04:54:38.312000Z",
     "gsd": 1.01,
-    "platform": "SS01",
+    "platform": "SSC1",
```

Comment:
The original metadata say SS01, but that's not allowed in the Planet extension (prefix always "SSC"). Is this wrong metadata on Planet's side or do we need to extend the pattern in the extension? To me it looks like all other files use SSC, also in this tree of files.

Reply:
PR opened for the extension: https://github.com/planetlabs/open-stac/pull/3#discussion_r1218061445

Will update the values here, too.